packages/react-components/src/components/ActionBar/ActionBar.module.scss

```diff
@@ -44,6 +51,108 @@ $base-class: 'action-bar';
 
   &__menu-wrapper {
     display: flex;
-    align-items: center;
+    flex: 0 0 70px;
+    padding: var(--spacing-2) var(--spacing-1);
+
+    &__button {
+      padding: 6px;
```

Comment:
spacing token?

Reply:
This is a value closely related to this component and its logic.